pkg/cmd/pr/shared/preserve_input.go

```diff
@@ -0,0 +1,52 @@
+package shared
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/cli/cli/pkg/iostreams"
+)
+
+func PreserveInput(io *iostreams.IOStreams, ims *IssueMetadataState, defs Defaults, doPreserve *bool) func() {
+	return func() {
+		if ims.Body == defs.Body && ims.Title == defs.Title {
```

Comment:
Considering that the metadata state has more information than just the body and title I don't know if this check against the default body and title is very valuable. I would say that in the cases of failure where we want to dump the output we should write to disk whatever information we have regardless of what it is.

Reply:
i want a check like this so if someone ctrl-c's immediately we don't litter the filesystem; but I'm realizing I should just augment it with `nil` checks for metadata.